models/_generate_layers.py

```diff
@@ -263,6 +266,14 @@ def _init_args(self):
           kind=inspect.Parameter.POSITIONAL_OR_KEYWORD),
         param_type_s="int",
         docstring="output dimension")
+    if self.layer_class in self._LayerClassesWithExplicitTarget:
+      self.params["target"] = LayerSignature.Param(
+        self,
+        inspect.Parameter(
+          name="target",
+          kind=inspect.Parameter.KEYWORD_ONLY),
+        param_type_s="LayerRef",
```

Comment:
I'm just realizing that `LayerRef` is the wrong type here. `str` would be correct.

Although I'm not sure whether it actually maybe should be a `LayerRef`. See my other comment.